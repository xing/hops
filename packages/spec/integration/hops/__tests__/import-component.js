import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import testRenderer from 'react-test-renderer';
import App from '../';

describe('ImportComponent', () => {
  it('should include the default mock result', () => {
    const app = testRenderer.create(
      <MemoryRouter>
        <App />
      </MemoryRouter>
    );

    expect(app).toMatchSnapshot();
  });
});
