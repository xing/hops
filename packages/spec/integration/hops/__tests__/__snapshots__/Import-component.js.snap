// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImportComponent should include the default mock result 1`] = `
<ImportComponentStub
  loader={[Function]}
  render={[Function]}
/>
`;
